### ISSUE TITLE
A Data Scientist at a retail company is using Amazon SageMaker to classify social media posts that mention the company into one of two categories: Posts that require a response from the company, and posts that do not. The Data Scientist is using a training dataset of 10,000 posts, each of which contain the timestamp, author, and full text of each post. However, the Data Scientist is missing the target labels that are required for training. Which approach can the Data Scientist take to create valid target label data? (Select TWO.)

### ISSUE THREAD

**Issue by epassaro** [epassaro/aws-mls-c01#7]
### A Data Scientist at a retail company is using Amazon SageMaker to classify social media posts that mention the company into one of two categories: Posts that require a response from the company, and posts that do not. The Data Scientist is using a training dataset of 10,000 posts, each of which contain the timestamp, author, and full text of each post. However, the Data Scientist is missing the target labels that are required for training. Which approach can the Data Scientist take to create valid target label data? (Select TWO.)

- [ ] Ask the social media handling team to review each post using Amazon SageMaker GroundTruth and provide the
label
- [ ] Use the sentiment analysis natural language processing library to determine whether a post requires a response
- [ ] Use Amazon Mechanical Turk to publish Human Intelligence Tasks that ask Turk workers to label the posts
- [ ] Use the a priori probability distribution of the two classes. Then, use Monte-Carlo simulation to generate the
labels
- [ ] Use K-Means to cluster posts into various groups, and pick the most frequent word in each group as its label

<details>
<summary><h3>Solución</h3></summary>

- [x] Ask the social media handling team to review each post using Amazon SageMaker GroundTruth and provide the
label
- [ ] Use the sentiment analysis natural language processing library to determine whether a post requires a response
- [x] Use Amazon Mechanical Turk to publish Human Intelligence Tasks that ask Turk workers to label the posts
- [ ] Use the a priori probability distribution of the two classes. Then, use Monte-Carlo simulation to generate the
labels
- [ ] Use K-Means to cluster posts into various groups, and pick the most frequent word in each group as its label

</details>

<details>
<summary><h3>Razonamiento</h3></summary>
 
Las dos primeras opciones son las que permiten obtener etiquetas de alta calidad a partir de la intervención humana, lo cual es crítico para entrenar un modelo de clasificación supervisado.

Esta pregunta no me gusta porque asume mucho contexto, ¿cómo puedo asumir que yo tengo suficiente autoridad como para recomendar esas opciones?

</details>
